src/test/java/org/broadinstitute/hellbender/utils/read/testers/SamFileTester.java

```diff
@@ -255,18 +262,22 @@ public void addMatePair(final String readName,
      */
     public void runTest() {
         final File input = createInputFile();
-
         output = new File(outputDir, "output.sam");
-        addArg("--INPUT", input.getAbsolutePath());
-        addArg("--OUTPUT", output.getAbsolutePath());
+        if (useStandardInputNames) {
```

Comment:
I don't like that we have to have this switch here.  Could you open a ticket to update tools so that it is unnecessary? Subtask of #133